pkg/cmd/clustertask/start_test.go

```diff
@@ -1623,7 +1623,9 @@ func Test_ClusterTask_Start_v1beta1(t *testing.T) {
 				if err == nil {
 					t.Errorf("Error expected here")
 				}
-				test.AssertOutput(t, tp.want, err.Error())
+				if !strings.HasPrefix(err.Error(), tp.want) {
+					t.Errorf("unexpected output:  expected prefix: %s, actual: %s", tp.want, err.Error())
+				}
```

Comment:
Maybe for continuity of the existing tests, it would make sense to add a field to `tp` (maybe a boolean named `hasPrefix`). If `hasPrefix` is true, this new check you have added takes place. If `hasPrefix` is false, then `test.AssertOutput(t, tp.want, err.Error())` is executed. 

Additionally, what if this is defined as a helper func in [helper.go](https://github.com/tektoncd/cli/blob/2d3875def20311c3b762f487cfe046a6448e685f/pkg/test/helpers.go) and followed a similar output format for errors as [`AssertOutput`](https://github.com/tektoncd/cli/blob/2d3875def20311c3b762f487cfe046a6448e685f/pkg/test/helpers.go#L49)?

What do you think?

Reply:
Sounds great! Let me fix the code quickly 😃